dask/array/routines.py

```diff
@@ -41,8 +42,10 @@
 
 
 @derived_from(np)
-def array(x, dtype=None, ndmin=None):
-    x = asarray(x)
+def array(x, dtype=None, ndmin=None, *, like=None):
+    if not _numpy_120:
```

Comment:
Similar comment here

Reply:
Also done in https://github.com/dask/dask/pull/8054/commits/37c85ad08dc9e4bc068f75ef837d5eadc6bbafce .